From 2a69a6e413c4aa5c7662f971e535d559d6013f7c Mon Sep 17 00:00:00 2001
From: Teo Hall <teo.hall@freescale.com>
Date: Thu, 24 Sep 2015 13:44:39 -0500
Subject: [PATCH] MLK-11262-4: ARM: imx: add m4_bootrom and m4_tcm to dts

add tcm and bootrom to device tree for m4 re-kick after DSM.

Signed-off-by: Teo Hall <teo.hall@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb-m4.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb-m4.dts b/arch/arm/boot/dts/imx7d-sdb-m4.dts
index 8586ed1..a6642a8 100644
--- a/arch/arm/boot/dts/imx7d-sdb-m4.dts
+++ b/arch/arm/boot/dts/imx7d-sdb-m4.dts
@@ -13,6 +13,11 @@
 		linux,usable-memory = <0x80000000 0x3ff00000>;
 		reg = <0x80000000 0x80000000>;
 	};
+
+	m4_tcm: tcml@007f8000 {
+		compatible = "fsl, m4_tcml";
+		reg = <0x007f8000 0x8000>;
+	};
 };
 
 &adc1 {
-- 
1.8.0

